lib/maintainers.nix

```diff
@@ -483,7 +483,7 @@
   oxij = "Jan Malakhovski <oxij@oxij.org>";
   paholg = "Paho Lurie-Gregg <paho@paholg.com>";
   pakhfn = "Fedor Pakhomov <pakhfn@gmail.com>";
-  palo = "Ingolf Wanger <palipalo9@googlemail.com>";
+  palo = "Ingolf Wanger <nixos@ingolf-wagner.de>";
```

Comment:
is that you? the key should be your github handle

Reply:
Right, damn. both emails belong to me, but the googlemail.com is my github handle. let me check if I can change that